hpOneView/common.py

```diff
@@ -1695,6 +1696,24 @@ def make_alertMap_dict(notes, etag, state='Active', user='None',
     }
 
 
+def make_migration_information(oaIpAddress, oaUsername, oaPassword, vcmUsername, vcmPassword, iloLicenseType='OneView',
```

Comment:
I like the idea of putting the builders as static methods in the resource classes, even the global method inside the module is not bad.